js/ui/popup.js

```diff
@@ -755,7 +755,7 @@ var Popup = Overlay.inherit({
     },
 
     _toggleSafariFullScreen: function(value) {
-        var toggleFullScreenBeforeShown = this._useFixedPosition() && value && !this.isReady();
+        var toggleFullScreenBeforeShown = this._useFixedPosition() && value && !this._isShown;
```

Comment:
this._currentVisible?

Reply:
The value of this variable is set to true before the animation starts. In this case, it does not seem to be suitable